model/foodEntry/index.js

```diff
@@ -18,6 +18,20 @@ class FoodEntryModel {
     return await addDoc(foodEntriesRef, foodObj);
   };
 
```

Comment:
Since getCurrentDayHistory always gets "today's history", there is no need to pass in a today argument. Adjust this function and the driver code to reflect this change.